iroh-relay/Cargo.toml

```diff
@@ -96,6 +96,7 @@ url = { version = "2.5", features = ["serde"] }
 webpki = { package = "rustls-webpki", version = "0.102" }
 webpki-roots = "0.26"
 data-encoding = "2.6.0"
+lru = "0.12.5"
```

Comment:
Purely a drive-by comment.  But I've had another reason to add an LRU cache to iroh-relay very recently.  So I think this dependency would make it here at some point.